Jenkinsfile.nightly

```diff
@@ -29,11 +57,12 @@ pipeline {
                 K8S_VERSION=1.9
                 MEMORY=4096
                 CPU=4
+                FAILFAST=setIfPR("true", "false")
```

Comment:
Why not `ispr` here? We seem to do this in a few places. If we have to use a function for these assignments it would be good to mention that in the description of the exported functions.